src/cron/personalizedDigest.ts

```diff
@@ -61,6 +62,8 @@ const cron: Cron = {
     // Make sure digest is sent at the beginning of the hour
     const timestamp = startOfHour(new Date());
 
+    const briefingUptime = await briefFeedClient.getBriefLastUpdate();
```

Comment:
at the end managed to fetch once per cron execution, and then check below for each row